src-tauri/src/main.rs

```diff
@@ -117,6 +120,14 @@ fn is_debug_mode() -> bool {
     return cfg!(debug_assertions);
 }
 
+#[allow(dead_code)]
+#[cfg(target_os = "linux")]
+#[tauri::command]
+/// Returns true if linux compatible
+fn linux_checks() -> bool {
+    return linux_checks_librs();
+}
```

Comment:
So uh, turns out this doesn't actually get run when clicking the button cause the `linux_check_function` is not added to [the invoke handler (linked here)](https://github.com/GeckoEidechse/FlightCore/blob/e29cf4bbdb0aa19978a560c2dc563c40a44fba13/src-tauri/src/main.rs#L76-L94) ^^

Reply:
And when added, it panics when clicking the button on Linux with:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseFloatError { kind: Invalid }', src/platform_specific/linux.rs:15:53
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

For reference, this is the version `ldd` prints when calling it manually from console on Ubuntu 22.04 LTS:

```
$ /bin/ldd --version
ldd (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
Copyright (C) 2022 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.
```

Reply:
It works fine on Arch, let me start a ubuntu container and try some stuff.

Reply:
I'm a bit stuck here, I can't think of a consistent way with my limited rust knowledge to find out where the version number actually is and then get just that in a var.

Reply:
Aight, I'll take a look when I get to it ^^

Reply:
Collected some outputs of `/bin/ldd --version` in #linux on Northstar Discord (only first line):

```
ldd (GNU libc) 2.35
```

```
ldd (Ubuntu GLIBC 2.35-0ubuntu3.1) 2.35
```

Reply:
More: [ldd.txt](https://github.com/GeckoEidechse/FlightCore/files/9775464/ldd.txt)

Reply:
I'll try to look at it soon, can't today or likely not tommorow though.